easypy/concurrency.py

```diff
@@ -332,7 +332,9 @@ def add_details(exc):
 
             with buff.indent(f"{{.__class__.__qualname__}} ({full_exc_num})", exc):
                 if isinstance(exc, MultiException):
-                    buff.extend(exc._get_buffer(exc_num_prefix=f"{full_exc_num}.", **kw))
+                    buff.extend(exc._get_buffer(exc_num_prefix=f"{full_exc_num}.",
+                                                unique_tbs_to_exc_num=unique_tbs_to_exc_num,
+                                                **kw))
```

Comment:
Don't align lines, always 4-space indents, break lines at parenthesis:
```suggestion
                    buff.extend(exc._get_buffer(
                        exc_num_prefix=f"{full_exc_num}.",
                        unique_tbs_to_exc_num=unique_tbs_to_exc_num,
                        **kw))
```